docs/04-how-we-work/tools/tickets-cards.md

```diff
@@ -25,12 +25,14 @@ Description: An epic often starts as the big picture and then the user stories f
 - Notes that explain how/where to start
 - Helps if another engineer has to pick up your ticket
 
-## Acceptance Tests
+## User Acceptance Tests (UAT)
```

Comment:
I wonder if we should add some more context here, or perhaps a less specific term? In the wider world UAT has often been a pretty waterfally multi-week/month process - https://huddle.eurostarsoftwaretesting.com/unconventional-wisdom-v16-user-uat-vs-user-story-acceptance-tests/ suggests a similar thing.